apparmor.d/profiles-m-r/mdevctl

```diff
@@ -11,6 +11,11 @@ profile mdevctl @{exec_path} {
   include <abstractions/base>
 
   @{exec_path} mr,
+  @{bin}/basename rix,
+  @{bin}/find rix,
+  @{bin}/getopt rix,
+  @{bin}/jq   rix,
+  @{bin}/sort rix,
```

Comment:
mdevctl is https://github.com/mdevctl/mdevctl. This is not a bash script. Why is this here?

Reply:
I can't find the original audit logs for this anymore, and a system reboot doesn't produce them either. I'll just remove this for now.